database/db.js

```diff
@@ -0,0 +1,11 @@
+let mysql = require('mysql');
+let sqlPassword = require('..config');  
+
+let connection = mysql.createConnection({
+    host: 'local host', 
```

Comment:
I think this needs to be "localhost" I've never tried to do "local host" but my gut says that might be bad?

Reply:
I've got this fixed for the next version. Will have to come w/ the next branch pull request because I did not branch originally and was just working from the master.